### PR TITLE
Use hie-bios 0.12

### PR DESCRIPTION
**Pull request by wz1000**

### CODE REVIEW

**Review by fendor — APPROVED** [review]
LGTM. thank you!

**Comment by wz1000** [comment]
@fendor @michaelpj can you please review the logging in Wrappers.hs?

File: ghcide/session-loader/Development/IDE/Session.hs [hunk 1]
```diff
@@ -725,7 +725,7 @@ cradleToOptsAndLibDir recorder cradle file = do
     case cradleRes of
         CradleSuccess r -> do
             -- Now get the GHC lib dir
-            libDirRes <- getRuntimeGhcLibDir cradle
+            libDirRes <- getRuntimeGhcLibDir (toCologActionWithPrio (cmapWithPrio LogHieBios recorder)) cradle
```
Comment (fendor) [inline ghcide/session-loader/Development/IDE/Session.hs:728]:
```suggestion
            libDirRes <- getRuntimeGhcLibDir logger cradle
```

logger is defined on line 723

**Review by fendor — APPROVED** [review]
Looks fine as is. However, according to some comments, we can simplify the logging now. Let's get this merged, and then we can look at simplifying the logging story.

**Review by michaelpj — APPROVED** [review]
Looks fine to me. I think the main knock-on change is that we can now finally purge `hslogger`.

**Pull request merged**